feature-libs/product-configurator/rulebased/components/restart-dialog/configurator-restart-dialog.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="cx-modal-container">
+  <ng-container *ngIf="dialogData$ | async as dialogData">
+    <ng-container *ngIf="product$ | async as product">
+      <div
+        class="cx-modal-content"
+        [cxFocus]="focusConfig"
+        (esc)="backToPDP(product)"
+      >
+        <div class="cx-dialog-header modal-header">
+          <div class="cx-dialog-title modal-title">
```

Comment:
Please, remove `cx-dialog-title` class. `modal-title` class is enough to style this dialog accordingly.
The goal is to hold HTML tree as simple as possible.